ogcapi-features/README.md

```diff
@@ -16,7 +18,7 @@
   - [Example Collection Endpoint](#example-collection-endpoint)
   - [Extensions](#extensions)
 
-*based on [**OGC API - Features - Part 1: Core**](https://www.ogc.org/standards/ogcapi-features)*
```

Comment:
Removed intentionally?

Reply:
Yes, since this is the first sentence in the Overview section.